strings/strings.ko-KR.json

```
@@ -9025,16 +9025,16 @@
   "wiki_challenges_modes_gravity_well_para4": "높은 연료 소모 말고도 생산을 보조하고 자재를 운송하려면 운반부를 고용해야 합니다.",
   "wiki_challenges_modes_gravity_well_para5": "운반부를 고용하지 못하게 되면 대부분의 생산량이 감소합니다.",
   "wiki_challenges_modes_gravity_well_para6": "더 많은 인원을 고용하고 더 많은 생산 시설을 운영할수록, 운반부도 더 많이 필요합니다.",
-  "wiki_challenges_scenarios_intro": "Scenarios",
+  "wiki_challenges_scenarios_intro": "시나리오",
   "wiki_challenges_scenarios_intro_para1": "%0 are challenges that place the player into more unique situations.",
   "wiki_challenges_scenarios_intro_para2": "They enforce use of all four available %0 and each have associated achievements that provide perks like %1 do.",
   "wiki_challenges_scenarios_intro_para3": "Unlike %0 and %1, only one %2 can be selected at a time during evolution.",
-  "wiki_challenges_scenarios_junker": "Genetic Dead End",
+  "wiki_challenges_scenarios_junker": "나락에 빠진 유전자",
```

```suggestion
  "wiki_challenges_scenarios_junker": "막장 유전자",
```
원문을 다시 보니 Genetic Dead End라고 하는데, 이 Dead End가 '끝에 다다른', '나아갈 수 없는' 이라는 뜻을 가집니다. 한국어로 가장 알맞은 번역은 '갈 데 까지 간 유전자'라든가 '막장 유전자'라고 번역할 수 있을텐데 이 표현은 어떨까요?

---

이미 번역되어 있는 부분이어서 기존 번역과 맞춘 것이고요.
변경하려면 게임 시작때 어느게 더 잘 어울리나 봐주시면 좋을거 같은데, 제 느낌은 바꿔서 더 좋아지는 거 같지는 않습니다